ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -97,6 +106,214 @@ tests:
+    - as: e2e-test
+      cli: latest
+      commands: make func-test OPENSHIFT_BUILD_NAMESPACE=
+      dependencies:
+      - env: OPERATOR_IMAGE
+        name: pipeline:cincinnati-operator
+      - env: RELATED_IMAGE_OPERATOR
+        name: pipeline:cincinnati-operator
+      - env: GRAPH_DATA_IMAGE
+        name: graphdata
+      - env: RELATED_IMAGE_OPERAND
+        name: cincinnati_operand_latest
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: generic-claim
+- as: operator-e2e-412
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
+  steps:
+    test:
+    - as: e2e-test
+      cli: latest
+      commands: |
+        make func-test \
+        RELATED_IMAGE_OPERAND=registry.redhat.io/openshift-update-service/openshift-update-service-rhel8@sha256:0e67daa2313ac7695b53de468ca0a540988befe1dfa5c9036eb05e21fd9f107a \
+        OPENSHIFT_BUILD_NAMESPACE=
+      dependencies:
+      - env: OPERATOR_IMAGE
+        name: pipeline:cincinnati-operator
+      - env: RELATED_IMAGE_OPERATOR
+        name: pipeline:cincinnati-operator
+      - env: GRAPH_DATA_IMAGE
+        name: pipeline:cincinnati-graph-data-container
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: generic-claim
+- as: operator-e2e-latest-osus-412
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
+  steps:
+    test:
+    - as: e2e-test
+      cli: latest
+      commands: make func-test OPENSHIFT_BUILD_NAMESPACE=
+      dependencies:
+      - env: OPERATOR_IMAGE
+        name: pipeline:cincinnati-operator
+      - env: RELATED_IMAGE_OPERATOR
+        name: pipeline:cincinnati-operator
+      - env: GRAPH_DATA_IMAGE
+        name: graphdata
+      - env: RELATED_IMAGE_OPERAND
+        name: cincinnati_operand_latest
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: generic-claim
+- as: operator-e2e-413
+  steps:
+    cluster_profile: aws-2
+    dependencies:
+      RELEASE_IMAGE_LATEST: release:thirteen
+    test:
+    - as: e2e-test
+      cli: latest
+      commands: |
+        make func-test \
+        RELATED_IMAGE_OPERAND=registry.redhat.io/openshift-update-service/openshift-update-service-rhel8@sha256:0e67daa2313ac7695b53de468ca0a540988befe1dfa5c9036eb05e21fd9f107a \
+        OPENSHIFT_BUILD_NAMESPACE=
+      dependencies:
+      - env: OPERATOR_IMAGE
+        name: pipeline:cincinnati-operator
+      - env: RELATED_IMAGE_OPERATOR
+        name: pipeline:cincinnati-operator
+      - env: GRAPH_DATA_IMAGE
+        name: pipeline:cincinnati-graph-data-container
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: hypershift-hostedcluster-workflow
```

Comment:
Moving from cluster-claims to HyperShift is interesting.  But before removing the WIP, we probably want to pick one approach and consolidate around that.